src/redux/actions/internalActions.js

```diff
@@ -373,4 +373,9 @@ export const setIsProgrammaticSearchFull = isProgrammaticSearchFull => ({
 export const setNoteTransformFunction = noteTransformFunction => ({
   type: 'SET_NOTE_TRANSFORM_FUNCTION',
   payload: { noteTransformFunction },
+});
+
+export const setEnableSnapMode = enable => ({
+  type: 'SET_ENABLE_SNAP_MODE',
```

Comment:
i think this is my first time setting up a redux state in this code base. let me know if i put it incorrectly